ParkingLot/ParkingBoy.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Dynamic;
+using System.Globalization;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace ParkingLot
+{
+    public class ParkingBoy
+    {
+        private Dictionary<Car, ParkingLot> parkedCars = new Dictionary<Car, ParkingLot>();
+        private List<ParkingLot> parkingLots;
+        private int leftPosition;
+        public ParkingBoy(List<ParkingLot> parkingLots)
+        {
+            this.parkingLots = parkingLots;
+            leftPosition = parkingLots.Count * parkingLots[0].Capacity;

```

Comment:
0 is a magic number? why the leftposition only gives available position of the first parking lot?